stonks-app/code/database.py

```diff
@@ -189,6 +193,6 @@ def add_user_purchase_data_to_db(passphrase, data):
 def delete_user_purchase_data(passphrase, id_):
     hash_ = hash_passphrase(passphrase)
     db = get_firestore_client()
-    docs = db.collection(collection_name).where('hash', '==', hash_).where('id', '==', id_).stream()
+    docs = db.collection('purchases').where('hash', '==', hash_).where('id', '==', int(id_)).stream()
```

Comment:
`doc` didn't seem to have a `delete()` method, had to call `.reference` first. As for the `int()`, `id_` is of numpy64 int type which wasn't accepted